### ISSUE TITLE
`now whoami` not working in version 15.4.0

### ISSUE THREAD

**Issue by Txarli** [vercel/vercel#2410]
In version 15.4.0 the command `now whoami` throws the next error:
```
Error! The specified file or directory "whoami" does not exist.
```
When executing `now --help` the output displays `whoami` as a valid subcommand but, when executing `now --help whoami`, it throws the next error:
```
Error! The "whoami" subcommand does not exist
```
If this feature has been removed from the client the `now --help` output should be updated. If not, it seems there is a problem with the command.

**Comment by v1rtl** [comment]
same

**Comment by TooTallNate** [comment]
Thanks for the report, fixed in #2411.
